stream-chat-android-client/src/main/java/io/getstream/chat/android/client/notifications/handler/ChatNotificationHandler.kt

```diff
@@ -306,9 +306,9 @@ public open class ChatNotificationHandler @JvmOverloads constructor(
         )
     }
 
-    public open fun getFirebaseInstanceId(): FirebaseInstanceId? =
+    public open fun getFirebaseInstallations(): FirebaseInstallations? =
```

Comment:
It's breaking change. I think we should deprecate it first

Reply:
Last version of FCM removed this class.
Because Firebase Dependencies can (and used to) be updated on the customer apps, we should try to use the most recent one that already removed this class.
For this concrete case I think we can bypass our "deprecation process"

Reply:
To deprecate it first, we must not update the FCM lib to the latest version because the `FirebaseInstanceId` is not available anymore in this release. On the other hand, we need to update FCM to the latest version urgently because some clients are using the latest version of Firebase which overrides our transitive dependency in SDK (this is due to the fact Firebase uses BOM). This results in `FirebaseInstanceId` not being resolved. We decided to make an exception this time and that's a priority to make our SDK compatible with the latest version of Firebase ASAP.